spec/operators/toPromise-spec.ts

```diff
@@ -34,6 +34,8 @@ describe('Observable.prototype.toPromise', () => {
     Observable.of(42).toPromise().then((x: number) => {
       expect(wasCalled).to.be.true;
       expect(x).to.equal(42);
+
+      delete __root__.Rx;
```

Comment:
It's satisfying mocha's global leak checking, if test cade modifies global object and does not clear mocha catches it. In this test case it does not affect functionality but added for strict teardown.